package.json

```diff
@@ -35,6 +41,12 @@
 	},
 	"homepage": "https://github.com/artykovdaniyar/node.js-mentoring-program#readme",
 	"dependencies": {
-		"nodemon": "^3.1.0"
+		"@types/jest": "^29.5.12",
+		"axios": "^1.6.8",
+		"jest": "^29.7.0",
+		"node-notifier": "^10.0.1",
```

Comment:
`node-notifier` causes problems on my side, so I just ignored it. Probably some cross platform issues. But keep in mind, even if it works on your side, it does not mean it will work on server.